# Goal
string world_path
string model_uri
---
# Result
---
# Feedback
